ingenialink/canopen/network.py

```diff
@@ -185,6 +194,8 @@ def __init__(
         baudrate: CAN_BAUDRATE = CAN_BAUDRATE.Baudrate_1M,
     ):
         super(CanopenNetwork, self).__init__()
+        if not platform.system() == "Linux" and device != CAN_DEVICE.SOCKETCAN.value:
+            raise ILError("In Linux machines, only the SOCKETCAN device can be used.")
```

Comment:
Why? Technically the other drivers can be used too.

Reply:
That's right. I'll remove this error.